.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
Is there any more granular permission that can be used here?

Reply:
Unfortunately, it seems the pull request permission is a catch all. Autosubmit seems to have this same issue where it requires write access to edit the labels on PRs.

I filed a support ticket with GitHub at https://support.github.com/ticket/enterprise/1732/2139514 to see if they can allow smaller ACLs on pull request.